tests/test_quantification_accuracy.py

```
@@ -345,13 +344,13 @@ def test_small_diann_maxlfq_produces_results(self):
         )
 
         assert len(result) > 0, "MaxLFQ should produce results"
-        assert "MaxLFQIntensity" in result.columns
-        assert result["MaxLFQIntensity"].notna().sum() > 0, "Should have non-NaN intensities"
-        assert (result["MaxLFQIntensity"] > 0).all(), "All intensities should be positive"
+        assert "Intensity" in result.columns
+        assert result["Intensity"].notna().sum() > 0, "Should have non-NaN intensities"
+        assert (result["Intensity"] > 0).all(), "All intensities should be positive"
 
         print(f"\n[Small DIA-NN Subset] MaxLFQ results:")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)